src/tests/test_api.py

```diff
@@ -162,6 +173,66 @@ def test_training_output():
         print(users)
         assert len(users) == len(src.User.get_all_usernames("", db))
 
+def test_profiles_output():
+    src.api.profileOutput(db)
+
+    filename = "{}MyCollege_profiles.txt".format(apiFilePathTests)
+    assert exists(filename) ==  True
+
+    with open(filename) as f:
+        testfile = f.readlines()
+        correctprofile = db.execute("SELECT * FROM profile")
+
+        for i in range(6):
+            assert testfile[i] == correctprofile[i] + '\n'
```

Comment:
correctprofile probably needs a [0] somewhere.
```suggestion
            assert testfile[i] == correctprofile[i][0] + '\n'
```